source/code/plugin/CAdvisorMetricsAPIClient.rb

```diff
@@ -73,6 +73,11 @@ def getNodeCapacityFromCAdvisor(winNode: nil)
       return getResponse(winNode, relativeUri)
     end
 
+    def getPodsFromCAdvisor(winNode: nil)
```

Comment:
Do we need this method? All this method is doing is passing a relative uri.. Why not just call the get response method like the rest of the calls?

Reply:
as discussed, the current implementation cleaner interms readability so followed the same pattern.